lib/smtlib_lexer.mll

```diff
@@ -5,7 +5,7 @@ open Smtlib_parser
 exception SyntaxError of string
 
 let keywords =
-  let tbl = Hashtbl.create 10 in
+  let tbl = Hashtbl.create 16 in
```

Comment:
If you have a look at the stdlib code from OCaml, you'll see that it always takes the next power of two so this saves some time.

Reply:
Did not know that, thanks you so much! :smiley: